### PR TITLE
[pyqt/zh-cn] PyQT Chinese translation

### PR DESCRIPTION
**Pull request by kdxcxs**
- [x] I solemnly swear that this is all original content of which I am the original author
- [x] Pull request title is prepended with `[language/lang-code]`
- [x] Pull request touches only one file (or a set of logically related files with similar changes made)
- [x] Content changes are aimed at *intermediate to experienced programmers* (this is a poor format for explaining fundamental programming concepts)
- [x] If you've changed any part of the YAML Frontmatter, make sure it is formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
  - [x] Yes, I have double-checked quotes and field names!

### CODE REVIEW

**Comment by mxschumacher** [comment]
Hey - I'm having a hard time reviewing this PR because I don't speak Mandarin. Do you know somebody who speaks the language and could give some feedback?

File: zh-cn/pyqt-cn.html.markdown [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    b.setText("Hello World!")
+	# 设置尺寸和位置
+    w.setGeometry(100, 100, 200, 50)
+    b.move(50, 20)
+	# 设置窗口的标题
+    w.setWindowTitle("PyQt")
+	# 将所有东西都显示出来
+    w.show()
+	# 完成所有设置后，执行我们要求的操作
+    sys.exit(app.exec_())
```
Comment (imlonghao) [inline zh-cn/pyqt-cn.html.markdown]: 制表符和空格混用

Comment (kdxcxs) [reply zh-cn/pyqt-cn.html.markdown]: 多谢,已修改

File: zh-cn/pyqt-cn.html.markdown [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+tool: PyQT
+filename: learnpyqt.py
+contributors:
+    - ["Nathan Hughes", "https://github.com/sirsharpest"]
+translators:
+    - ["kdxcxs", "https://github.com/kdxcxs"]
+lang: zh-cn
+---
+
+**Qt** 是一个著名的用C++实现的跨平台软件开发框架。只需更改少量甚至有时候不需要更改代码就能在多个软硬件平台上运行同时拥有原生应用程序的功能和速度。
```
Comment (lsvih) [inline zh-cn/pyqt-cn.html.markdown]:
```suggestion
**Qt** 是一个用 C++ 实现的著名的跨平台软件开发框架。只需对代码进行少许改动甚至无需改动，就能让 Qt 编写的应用程序在多个软硬件平台上运行，并拥有原生应用程序的功能和速度。
```

File: zh-cn/pyqt-cn.html.markdown [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+
+这是[Aleksey Kholovchuk](https://github.com/vortexxx192)对QT的C++简介的改编，用pyqt实现了一些相同功能的代码！
```
Comment (imlonghao) [inline zh-cn/pyqt-cn.html.markdown]: 中文和英文之间最好有空格，连接前后也是

**Comment by keithnull** [comment]
@mxschumacher 
As a native Chinese speaker, maybe I can help?

As far as I'm concerned, this translation's quality is 6/10. It's basically correct but not very proper in grammar.

File: zh-cn/pyqt-cn.html.markdown (hunk 3 above)
Comment (lsvih) [inline zh-cn/pyqt-cn.html.markdown]:
```suggestion
以下内容改编自 [Aleksey Kholovchuk](https://github.com/vortexxx192) 编写的 C++ 版 QT 简介，并用 pyqt 重构原文代码，实现了部分相同的功能。
```

File: zh-cn/pyqt-cn.html.markdown [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+```python
+import sys
+from PyQt4 import QtGui
+    
+def window():
+    # 创建应用对象
+    app = QtGui.QApplication(sys.argv)
+    # 创建一个widget，label将会被放置在里面
```
Comment (lsvih) [inline zh-cn/pyqt-cn.html.markdown]:
```suggestion
    # 创建一个 widget，并将 label 置于其内
```

File: zh-cn/pyqt-cn.html.markdown [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+    w = QtGui.QWidget()
+    # 在widget中添加一个label
```
Comment (lsvih) [inline zh-cn/pyqt-cn.html.markdown]:
```suggestion
    # 在 widget 中添加一个 label
```

File: zh-cn/pyqt-cn.html.markdown [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+    b = QtGui.QLabel(w)
+    # 设置label的文字
```
Comment (lsvih) [inline zh-cn/pyqt-cn.html.markdown]:
```suggestion
    # 设置 label 的文字
```

File: zh-cn/pyqt-cn.html.markdown [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+    b.setText("Hello World!")
+    # 设置尺寸和位置
```
Comment (lsvih) [inline zh-cn/pyqt-cn.html.markdown]:
```suggestion
    # 设置 widget 的尺寸和位置
```

File: zh-cn/pyqt-cn.html.markdown [hunk 8]
```diff
@@ -0,0 +1,78 @@
...
+    w.setGeometry(100, 100, 200, 50)
+    b.move(50, 20)
+    # 设置窗口的标题
+    w.setWindowTitle("PyQt")
+    # 将所有东西都显示出来
```
Comment (lsvih) [inline zh-cn/pyqt-cn.html.markdown]:
```suggestion
    # 显示设置好的全部内容
```

File: zh-cn/pyqt-cn.html.markdown [hunk 9]
```diff
@@ -0,0 +1,78 @@
...
+    w.show()
+    # 完成所有设置后，执行我们要求的操作
+    sys.exit(app.exec_())
+if __name__ == '__main__':
+    window()
+```
+
+为了获得pyqt中的一些更高级的功能，我们需要开始研究构建其他元素。在这里，我们展示了如何使用弹出对话框，该对话框对于要求用户确认操作或提供信息很有用。
```
Comment (lsvih) [inline zh-cn/pyqt-cn.html.markdown]:
```suggestion
为了了解 pyqt 中的一些更高级的功能，我们需要研究如何构建其他元素。下文展示了如何使用弹出对话框，弹出对话框常用于要求用户确认操作或提供信息。
```

File: zh-cn/pyqt-cn.html.markdown [hunk 10]
```diff
@@ -0,0 +1,78 @@
...
+
+```Python 
+import sys
+from PyQt4.QtGui import *
+from PyQt4.QtCore import *
+def window():
+    app = QApplication(sys.argv)
+    w = QWidget()
+    # 创建一个按钮并附加到widget w
```
Comment (lsvih) [inline zh-cn/pyqt-cn.html.markdown]:
```suggestion
    # 创建一个按钮并附加到 widget w
```

File: zh-cn/pyqt-cn.html.markdown [hunk 11]
```diff
@@ -0,0 +1,78 @@
...
+    b = QPushButton(w)
+    b.setText("Press me")
+    b.move(50, 50)
+    # 当按钮b被点击时调用下面这个函数
```
Comment (lsvih) [inline zh-cn/pyqt-cn.html.markdown]:
```suggestion
    # 当按钮 b 被点击时调用 showdialog 函数
```

File: zh-cn/pyqt-cn.html.markdown [hunk 12]
```diff
@@ -0,0 +1,78 @@
...
+    # 注意函数调用时没有“()”，这样函数就能以对象的方式传入而非调用它所得到的返回值
+    # 更多关于pyqt函数调用、传参等的内容见pyqt的信号机制
```
Comment (lsvih) [inline zh-cn/pyqt-cn.html.markdown]:
```suggestion
    # 更多关于 pyqt 函数调用、传参等的内容请参见 pyqt 的信号机制
```

File: zh-cn/pyqt-cn.html.markdown [hunk 13]
```diff
@@ -0,0 +1,78 @@
...
+    b.clicked.connect(showdialog)
+    w.setWindowTitle("PyQt Dialog")
+    w.show()
+    sys.exit(app.exec_())
+    
+# 此函数将会创建一个带有按钮的对话框窗口
+# 当按钮被点击时会退出这个程序
```
Comment (lsvih) [inline zh-cn/pyqt-cn.html.markdown]:
```suggestion
# 当按钮被点击时会退出本程序
```

File: zh-cn/pyqt-cn.html.markdown (hunk 3 above)
Comment (kdxcxs) [reply zh-cn/pyqt-cn.html.markdown]: 多谢,已整改

**Comment by lsvih** [comment]
@kdxcxs Perhaps suggestions https://github.com/adambard/learnxinyminutes-docs/pull/3855#discussion_r380439138  , https://github.com/adambard/learnxinyminutes-docs/pull/3855#discussion_r380439004  and https://github.com/adambard/learnxinyminutes-docs/pull/3855#discussion_r380437551  could be considered again?

File: zh-cn/pyqt-cn.html.markdown [hunk 14]
```diff
@@ -0,0 +1,78 @@
...
+from PyQt4 import QtGui
+    
+def window():
+    # 创建应用对象
+    app = QtGui.QApplication(sys.argv)
+    # 创建一个 widget，作为 label 的父控件
+    w = QtGui.QWidget()
+    # 在 widget 中添加一个 label 子控件
+    b = QtGui.QLabel(w)
+    # 设置 labe l的文字
```
Comment (lsvih) [inline zh-cn/pyqt-cn.html.markdown]:
```suggestion
    # 设置 label 的文字
```

Comment (kdxcxs) [reply zh-cn/pyqt-cn.html.markdown]: thx

**Comment by kdxcxs** [comment]
@lsvih Thanks so much.

**Comment by mxschumacher** [comment]
Hey @keithnull - if you could provide suggestions on how we can crank it up to 8/10, that would be fantastic!

**Comment by keithnull** [comment]
@mxschumacher 

After @kdxcxs @lsvih @imlonghao 's improvements, I'm sure that it's above 8/10 now.

**Comment by mxschumacher** [comment]
Okay - thanks everybody!

**Pull request merged**